node_modules/gh-rest/lib/rest/auth.js

```diff
@@ -49,4 +49,62 @@ module.exports = function(client) {
     client.auth.logout = function(callback) {
         client._request('/api/auth/logout', 'POST', null, null, callback);
     };
+
+    /**
+     * Redirect a user from an application to the SP
+     *
+     * @param  {String}         redirectUrl         The URL where the user should be redirect to once he succesfully authenticates
+     * @param  {Function}       callback            Standard callback method
+     * @param  {Object}         callback.err        An error object, if any
+     * @param  {String}         callback.body       The response body
+     * @param  {Object}         callback.response   The HTTP response object
+     */
```

Comment:
Should have the `@see gh-auth/lib/rest.js for more information` here and throughout